### ISSUE TITLE
weekday conditional off by one day

### ISSUE THREAD

**Issue by karlfast** [ttscoff/bunch#284]
The weekday conditional only works on the day before the actual day. 

I'm submitting this on Monday morning (about 10am CST) and have just tested the following examples. The conditional works when I make it about Sunday (ie: the day before). It fails when I make it about Monday (ie: the day it should trigger)

This bunch conditional does NOT work when it is Monday

```
if weekday is Monday
    Messages
end
```

These bunches DO work on Monday.

```
if weekday is Sunday
    Messages
end

if weekday is before Monday
    Messages
end

```

**Comment by ttscoff** [comment]
I'm testing on tuesday, and `if weekday is tuesday` is functioning fine 
for me. What Time Zone is your system set for? What language? I'm not 
sure what could be affecting it like that.

-Brett

**Comment by karlfast** [comment]
I'm not sure why either. 

Yesterday (Monday) and last week it would not work. I tried this many times over several days. I only posted yesterday given how many times this failed for me.

But today it worked. All that happened was I saw your reply, opened my bunch, and changed Monday to Tuesday.

About settings. Nothing special here. 

Time Zone: CST
Language: English (US)

I wondered if maybe I had First Day of Week set to Monday. In Busycal, yes. But not in System Prefs > Language & Region > Advanced. There the week starts on Sunday.

CONSIDER THIS SOLVED/CLOSED for now. I will followup if the problem returns or I discover something.
